en/lessons/basics/date-time.md

```diff
@@ -92,8 +90,13 @@ It does not have the limitations noted in `NaiveDateTime`: it has both time and
 But be aware about timezones. The official docs state:
 
 ```
-You will notice this module only contains conversion functions as well as functions that work on UTC.
-This is because a proper DateTime implementation requires a time zone database which currently is not provided as part of Elixir.
+Many functions in this module require a time zone database. By default, it uses the default time zone database returned by Calendar.get_time_zone_database/0, which defaults to Calendar.UTCOnlyTimeZoneDatabase which only handles "Etc/UTC" datetimes and returns {:error, :utc_only_time_zone_database} for any other time zone.
```

Comment:
Instead of the three backtics to denote a quote, markdown recommends using the `>` carrot. Then, within that block quote, let's use the single backtic to wrap `Calendar.get_time_zone_database/0`, `Calendar.UTCOnlyTimeZoneDatabase` and `{:error, :utc_only_time_zone_database}` and any other code blocks.